### PR TITLE
Test plugin with multiple versions of nexus

### PR DESCRIPTION
**Pull request by talebipour**
+ Provide feature to test plugin with multiple versions of nexus
 + Test plugin with nexus 3.24.0

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -108,6 +114,8 @@
                 <extensions>true</extensions> <!-- Support Bundle Packaging -->
                 <configuration>
                     <instructions>
+                        <!-- Relax dependency version in order to allow run inside multiple versions of nexus -->
+                        <_consumer-policy>$(version;==)</_consumer-policy>
```
Comment (MSDehghan) [inline pom.xml]: i think it should be like this:
```suggestion
                        <_consumer-policy>$${version;==}</_consumer-policy>
```

File: src/test/java/ir/sahab/nexus/plugin/tag/IntegrationTest.java [hunk 2]
```diff
@@ -268,6 +272,7 @@ private void assertDefinitionEquals(TagDefinition expected, Tag actual) {
 
     @AfterClass
     public static void tearDown() {
+        repositoryRule.after();
```
Comment (MSDehghan) [inline src/test/java/ir/sahab/nexus/plugin/tag/IntegrationTest.java]: Why not define RepositoryRule as a ClassRule??

File: pom.xml (hunk 1 above)
Comment (talebipour) [reply pom.xml]: Done

File: src/test/java/ir/sahab/nexus/plugin/tag/IntegrationTest.java (hunk 2 above)
Comment (talebipour) [reply src/test/java/ir/sahab/nexus/plugin/tag/IntegrationTest.java]: It's already a rule, I was testing sth and I forgot to remove it.

**Review by MSDehghan — APPROVED** [review]

**Review by MSDehghan — APPROVED** [review]
it's OK

**Pull request merged**